docs/images/snippets.md

```diff
@@ -47,26 +47,23 @@ fn QuickSort[T:! Comparable & Movable](s: Slice(T)) {
 
 ```cpp
 // C++:
-#include <math.h>
-#include <iostream>
-#include <span>
-#include <vector>
+import std;
 
 struct Circle {
-  float r;
+  std::float32_t r;
 };
 
 void PrintTotalArea(std::span<Circle> circles) {
-  float area = 0;
-  for (const Circle& c : circles) {
-    area += M_PI * c.r * c.r;
+  std::float32_t area = 0;
+  for (Circle c : circles) {
```

Comment:
I think the `const&` here is a slightly better match for the Carbon code, though neither is exactly right -- the Carbon code doesn't force a copy to be made, but it has value semantics rather than reference semantics.

Reply:
Thanks for the feedback. Agree that `const&` is a more faithful approximation